packages/@okta/vuepress-site/guides/sign-users-out/sign-out-of-okta/aspnet/remotesignout.md

```diff
@@ -0,0 +1,26 @@
+Open the controller where you will handle the Okta sign out process and add an `OktaSignOut` method. This time you need to also sign the user out of the OIDC middleware:
```

Comment:
Does aspnet not require you to register the logout redirect URI with Okta? The aspnetcore instructions cover it but these don't.

Reply:
Correct, aspnet doesn't require it.